lib/PuppeteerSharp/ChromeTargetManager.cs

```diff
@@ -137,29 +137,12 @@ private void StoreExistingTargetsForInit()
 
         private void OnMessageReceived(object sender, MessageEventArgs e)
         {
-            Action<Exception> handleException = (Exception ex) =>
-            {
-                var message = $"Browser failed to process {e.MessageID}. {ex.Message}. {ex.StackTrace}";
-                _logger.LogError(ex, message);
-                _connection.Close(message);
-            };
-
             try
             {
                 switch (e.MessageID)
                 {
                     case "Target.attachedToTarget":
-                        Task.Run(async () =>
-                        {
-                            try
-                            {
-                                await OnAttachedToTarget(sender, e.MessageData.ToObject<TargetAttachedToTargetResponse>(true)).ConfigureAwait(false);
-                            }
-                            catch (Exception ex)
-                            {
-                                handleException(ex);
-                            }
-                        });
+                        _ = OnAttachedToTargetHandlingExceptions(sender, e.MessageID, e.MessageData.ToObject<TargetAttachedToTargetResponse>(true)).ConfigureAwait(false);
```

Comment:
You don't need `ConfigureAwait(false);` if you are dropping the `Task`.

```suggestion
                        _ = OnAttachedToTargetHandlingExceptions(sender, e.MessageID, e.MessageData.ToObject<TargetAttachedToTargetResponse>(true));
```